src/main/java/htsjdk/samtools/seekablestream/SeekableStream.java

```diff
@@ -89,6 +89,28 @@ public void readFully(byte b[]) throws IOException {
         }
     }
 
+    /**
+     * The return value of this method is unusable for any purpose, and we are only implementing it
+     * because certain Java classes like {@link java.util.zip.GZIPInputStream} incorrectly rely on
+     * it to detect EOF
+     *
+     * <p>If {@code eof() == true}, 0 bytes are available. Otherwise, available bytes are the
+     * difference between the length of the stream ({@link #length()}) and the current position
+     * ({@link #position()}.
+     *
+     * @return {@code 0} if the end of the file has been reached or the length cannot be determined;
+     * number of bytes remaining in the stream otherwise.
+     */
+    @Override
+    public int available() throws IOException {
```

Comment:
This change is altered in...

Reply:
@magicDGS your comment is unclear....

Reply:
Sorry, I think that I submitted the review from the phone without reviewing. What I meant is that this was already implemented in the master branch (I guess that at that time it was a PR)